tiledb/api/src/array/mod.rs

```diff
@@ -814,7 +925,7 @@ pub mod tests {
                 .build()
         };
 
-        let s: Schema = SchemaBuilder::new(context, ArrayType::Sparse, d)
+        let s: Schema = SchemaBuilder::new(context, ArrayType::Dense, d)
```

Comment:
This seemed unintentional to me. The function is named "create_quickstart_dense".

Reply:
Everything seems to pass with the change. I can introduce a parameter to optionally create dense or sparse. Does that work for you?

Reply:
If everything passes I don't have a preference. As long as the function name isn't lying :)

Reply:
Ok. I will leave as above (use Dense)